vng_api_common/audittrails/constants.py

```diff
@@ -0,0 +1,11 @@
+from django.utils.translation import ugettext_lazy as _
+
+from djchoices import ChoiceItem, DjangoChoices
+
+
+class AuditTrailAction(DjangoChoices):
+    create = ChoiceItem('create', _("De resource is aangemaakt"))
+    get = ChoiceItem('get', _("De gegevens van de resource zijn opgehaald"))
```

Comment:
Volgens mij moet dit worden opgesplitst in `list` en `retrieve`.

Zie ook: https://www.django-rest-framework.org/api-guide/viewsets/#viewset-actions